### PR TITLE
Add intro to Docker network.

### PR DESCRIPTION
**Pull request by xavierlepretre**
This PR adds content to ease the student to the use of Docker Compose.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [ ] Small content fixes 
* [x] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

**Comment by AndrewB9lab** [comment]
Language review on "Docker network intro" content completed.

**Review by AndrewB9lab — APPROVED** [review]
Language review completed _and my approval to continue correctly registered_ ... thanks, Xavier :-)

File: hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md [hunk 1]
```diff
@@ -600,12 +602,115 @@ SystemInfo
...
+To run the checkers chain with Ignite CLI you have the choice of two Docker images:
+
+1. You can use the one published by Ignite themselves, for version `0.22.1`: [`ignitehq/cli:0.22.1`](https://hub.docker.com/layers/ignitehq/cli/0.22.1/images/sha256-8e2f353f943227488f966dd02558b718766a17dd8b611bccd2789facdceef0cf). This may be faster the first time you run it, but can become annoying if you plan on doing it many times as it will download the Go dependencies every time.
+2. You can build it yourself from the checkers [`Dockerfile-ubuntu`](https://github.com/cosmos/b9-checkers-academy-draft/blob/main/Dockerfile-ubuntu), with the command:
+
+    ```sh
+    $ docker build -f Dockerfile-ubuntu . -t checkers_i
+    ```
+
+    This is the preferred method if you plan on using the image many times, as it downloads all Go dependencies once.
```
Comment (samricotta) [inline hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md:620]: is this tab intentional or should i just be a space

**Review by samricotta — APPROVED** [review]
lgtm ! other than the one nit

File: hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md (hunk 1 above)
Comment (xavierlepretre) [reply hands-on-exercise/3-cosmjs-adv/1-cosmjs-objects.md:620]: It is intentional, in order to have this paragraph part of the indented numbered list item 2.

**Review by julienrbrt — APPROVED** [review]
ACK

**Review by coldice — APPROVED** [review]

**Pull request merged**